### ISSUE TITLE
Question about configuration setup

### ISSUE THREAD

**Issue by ztqwerty** [bilylee/SiamFC-TensorFlow#17]
Hi Bily大神，

I have a question about your configuration setup. In `MODEL_CONFIG`, there is an item called `adjust_response_config`, and you added a comment in `siamese_model.py`:

> Adjust score, this is required to make training possible 

Can you please provide some more explanation on the principle / reason behind that? Whether and when is it necessary to adjust the `train_bias` and `scale` parameters?

**Comment by bilylee** [comment]
Hi,

Score adjustment is simply normalizing response to the range (e.g. [-10, 10]) where sigmoid is not saturated. Without it, the scale of the response will be on the order of 1e3 and the gradients vanish.

**Comment by ztqwerty** [comment]
Thanks for the quick response. Appreciate it!
